src/ui/Quickview/Quickview.ts

```diff
@@ -250,30 +250,7 @@ export class Quickview extends Component {
      */
     tooltipPlacement: ComponentOptions.buildCustomOption<ValidTooltipPlacement>((value: ValidTooltipPlacement) => value, {
       defaultValue: 'bottom'
-    }),
-
-    /**
-     * Specifies the HTMLElement where the Modal Box will be created when opening a QuickView. You can
-     * either specify a CSS selector or pass an HTMLElement in the options when calling Coveo.init.
-     *
-     * Default value is `element.ownerDocument.body`.
-     *
-     * **Example in attribute:**
-     * ```html
-     * <div class="CoveoQuickview" data-modal-container="#my-modal-container"></div>
-     * ```
-     *
-     * **Example in init options:**
-     * ```javascript
-     * var myContainer = document.getElementById('my-modal-container');
-     * Coveo.init(root, {
-     *   Quickview: {
-     *     modalContainer: myContainer
-     *   }
-     * });
-     * ```
-     */
-    modalContainer: ComponentOptions.buildSelectorOption({ defaultFunction: element => element.ownerDocument.body })
```

Comment:
My concern around remove this option is whether other clients are now depending on it outside of ServiceNow.
The safest option would be to support both, but I guess we could bite the bullet. If someone opens a case, we'll direct them to the new option.

Reply:
Yeah... that's kind of eehhhhhh that we remove options. It means we did a bad job initially when that option was added to determine what was actually needed.

That's not backward compatible.

But I agree with Sami, lets *hope* that nobody was using that, lol.

Reply:
I see it's quite a recent PR https://github.com/coveo/search-ui/pull/1489. Maybe only 1 release contains it and like you say it was geared towards the SN integration. It wouldn't be a "big deal" to remove it in this case, but usually try not to break previous versions.